package.json

```diff
@@ -5,7 +5,9 @@
   "scripts": {
     "build": "ember build",
     "start": "ember server",
-    "test": "ember try:each"
+    "test": "npm run test-lib && npm run test-ember",
+    "test-ember": "ember try:each",
+    "test-lib": "./node_modules/tape/bin/tape tests-node/**/*-test.js"
```

Comment:
Can revert this, I added this before npm installing and realized it was user error